docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -358,6 +358,52 @@ Networking
 
 In general, if you want to use Airflow locally, your DAGs may try to connect to servers which are running on the host. In order to achieve that, an extra configuration must be added in ``docker-compose.yaml``. For example, on Linux the configuration must be in the section ``services: airflow-worker`` adding ``extra_hosts: - "host.docker.internal:host-gateway"``; and use ``host.docker.internal`` instead of ``localhost``. This configuration vary in different platforms. Please check the Docker documentation for `Windows <https://docs.docker.com/desktop/windows/networking/#use-cases-and-workarounds>`_ and `Mac <https://docs.docker.com/desktop/mac/networking/#use-cases-and-workarounds>`_ for further information.
 
+Debug Airflow inside docker container using Pycharm
+===================================================
+Prerequisites: Create a project in **PyCharm** and download the `docker-compose.yaml <{{ doc_root_url }}docker-compose.yaml>`__.
```

Comment:
I'm not sure why but locally URI doesn't rendered correctly: `http://127.0.0.1:8000/docs/apache-airflow/stable/howto/docker-compose/%7B%7Bdoc_root_url%7D%7Ddocker-compose.yaml`

Reply:
I copy `docker-compose.yaml <{{ doc_root_url }}docker-compose.yaml>`__. from https://github.com/apache/airflow/blob/ca11b7af239c90f6d7c00ed5116fdd08fda1a5ab/docs/apache-airflow/howto/docker-compose/index.rst?plain=1#L77

Reply:
Some magic of jinja templating doesn't work with sphinx 🤔 It works well before in other place, however in this place it keep unchanged even thought it exatly the same as it defined in `Fetching docker-compose.yaml`

Reply:
Ahhhh... figure it out

```suggestion
.. jinja:: quick_start_ctx

    Prerequisites: Create a project in **PyCharm** and download the `docker-compose.yaml <{{ doc_root_url }}docker-compose.yaml>`__.

```

![image](https://github.com/apache/airflow/assets/3998685/38878d59-808a-4a7e-9c23-294b8837a3ef)

Reply:
OH I see, this template only works in the Jinja context. I will add ``.. jinja:: quick_start_ctx``.

Reply:
I will work on these changes in my spare time, including the parts that failed the static checks. However, it is late at night for me, so I need to rest first. You can go ahead and work on other things.